core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletPort.scala

```diff
@@ -50,6 +50,22 @@ trait Inlet extends StreamletPort
  */
 trait Outlet extends StreamletPort
 
+/**
+ * A default error handler. This error handler just logs bad message and skips them.
+ *
+ */
+object CodecInlet {
+
+  val logger = LoggerFactory.getLogger(this.getClass)
+
+  def logAndSkip[T](message: Array[Byte], cause: Throwable): Option[T] = {
+    logger.error(s"Input data $message can not be transformed and will be skipped")
+    logger.error(s"Data decoding error ${cause.getMessage}")
```

Comment:
logger.error("Data decoding error", cause)

Reply:
wouldn't it be better to keep the getMessage? @RayRoestenburg  
`logger.error("Data decoding error, skipping message", cause.getMessage)`